Documentation/Blazorise.Docs/Pages/Blog/2024-07-19_HowToStyleBlazoriseComponents/Index.md

```diff
@@ -0,0 +1,162 @@
+﻿---
+title: How to style Blazorise components
+description: Discover
+permalink: /blog/how-to-style-blazorise-components
+canonical: /blog/how-to-style-blazorise-components
+image-url: /img/blog/2024-07-19/how-to-style-blazorise-components.png
+image-title: How to style Blazorise components
+author-name: Giorgi
+author-image: giorgi
+posted-on: Jul 19th, 2024
+read-time: 7 min
+---
+
+# Styling Blazorise components
+
+Hello! today we are going to go over how to style Blazorise components!
```

Comment:
Hello! Seems to informal, and we usual don't start with an informal greeting, we go straight to the point with the intro.
**today** should be capitalized.
I'd probably go over for a more succinct first paragraph like : 
"Learn how to style Blazorise components!"

**your know** => you know

"is a framework-agnostic library" => is a very generic wording, we are assuming every reader we'll immediately understand that we are talking about how Blazorise provides such an abstraction over what css framework is used, that pretty much any implementation can be made or plugged in. I wouldn't say that's it's agnostic per say, as it does need some kind of implementation to work correctly, but that the flexibility is there?

Anyway, my point is that we should be explicit that we're talking about css / ui framework.